File: src/ClientApp/src/app/pages/partners/view-models/partner-list-item.view-model.ts
export interface PartnerListItemViewModel {
    id: string;
    code: string;
    name: string;
    partnerTypeId: string;
    partnerTypeName: string;
    countryId: string;
    countryName: string;
    contactPerson?: string;
    phoneNumber?: string;
    email?: string;
    address?: string;
    remarks?: string;
    productIds: string[];
    isActive: boolean;
    createdAt: string;
}

File: src/API/src/Modules/Partners/Partners.Application/ViewModels/PartnerListItemViewModel.cs
namespace Partners.Application.ViewModels
{
    public class PartnerListItemViewModel
    {
        public Guid Id { get; set; }
        public string Code { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public Guid PartnerTypeId { get; set; }
        public string PartnerTypeName { get; set; } = string.Empty;
        public Guid CountryId { get; set; }
        public string CountryName { get; set; } = string.Empty;
        public string? ContactPerson { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Email { get; set; }
        public string? Address { get; set; }
        public string? Remarks { get; set; }
        public List<Guid> ProductIds { get; set; } = new();
        public bool IsActive { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}
